src/test/groovy/ratpack/zipkin/ServerTracingModuleSpec.groovy

```diff
@@ -368,4 +371,49 @@ class ServerTracingModuleSpec extends Specification {
 		then:
 			assertThat(reporter.getSpans()).isNotEmpty()
 	}
+
+	def 'Should nest client span and server span should have the same trace id'() {
+		given:
+			def webServer = new MockWebServer()
+			webServer.start()
+			webServer.enqueue(new MockResponse().setResponseCode(200))
+			def url = webServer.url("/")
```

Comment:
For the client tests I would imagine we would just instantiate ZipkinHttpClientImpl manually like I did for the brave integration tests.  For example:
https://github.com/hyleung/ratpack-zipkin/blob/master/src/test/java/brave/http/ITTracingFeature_Client.java#L39

I'm not advocating that we should tackle this now though :)  The fixes you have here are more important to have merged.